src/grid_fields.f90

```diff
@@ -50,7 +50,7 @@ subroutine w_alloc
 
  if(LPf_ord >0)then
   cfl_loc=cfl
-  if(ndim >1)cfl_loc=cfl*yx_rat/sqrt(yx_rat*yx_rat+1.)
+  if(ndim >1) cfl_loc=cfl*yx_rat/sqrt(yx_rat*yx_rat+ndim-1.)
```

Comment:
@cenit @framira :: Myself and @framira were discussing that to improve the user-friendly experience we should change the line 53 in 10-12 lines of codes with a less compact definition for 1,2 and 3 D. It will be longer but easier to be read.